pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -308,12 +308,15 @@ func startVirtlogdLogging(stopChan chan struct{}, domainName string, nonRoot boo
 
 			scanner := bufio.NewScanner(file)
 			scanner.Buffer(make([]byte, 1024), 512*1024)
-			for scanner.Scan() {
-				log.LogQemuLogLine(log.Log, scanner.Text())
-			}
-
-			if err := scanner.Err(); err != nil {
-				log.Log.Reason(err).Error("failed to read virtlogd logs")
+			for {
+				for scanner.Scan() {
+					log.LogQemuLogLine(log.Log, scanner.Text())
+				}
+				if err := scanner.Err(); err != nil {
+					log.Log.Reason(err).Error("failed to read virtlogd logs")
+					break
+				}
+				time.Sleep(100 * time.Millisecond)
```

Comment:
From what I can gather, the only way `Scan()` will return false without setting an error is if we reach EOF.
From that point on, I don't think we'll ever read anything from `scanner` again, or did I miss something?
Maybe instead of continuing on `err == nil` we want to continue on `io.ErrNoProgress`?
Either way, this seems tricky enough to warrant a unit test.